src/sound.js

```diff
@@ -216,26 +217,67 @@ export default class Sound {
   }
 
   playExplosion() {
-    this.playOneShot(coin, -16)
+    this.playOneShot(bubble, -36, { playbackRate: 2.3 })
+    this.playOneShot(bubble, -36, { playbackRate: 4.5 })
+    this.playOneShot(bubble, -16, { playbackRate: 0.2 })
+    setTimeout(() => {
+      this.playOneShot(bubble, -26, { playbackRate: 1 })
+      this.playOneShot(bubble, -26, { playbackRate: 5.5 })
+    }, 100)
+    setTimeout(() => {
+      this.playOneShot(bubble, -26, { playbackRate: 2.3 })
+      this.playOneShot(bubble, -26, { playbackRate: 5.5 })
+    }, 300)
   }
 
-  async playOneShot(url, volume) {
+  async playOneShot(url, volume, opts = false) {
     this.oneShots = this.oneShots || {}
-    if (!this.oneShots[url]) {
-      this.oneShots[url] = new Player({
+    const key = `${url}-${volume}-${opts && JSON.stringify(opts)}`
+    if (!this.oneShots[key]) {
+      this.oneShots[key] = new Player({
         url,
-        volume
+        volume,
+        ...opts
       }).toDestination()
     }
 
     // play if it's been loaded or loads quickly, otherwise load and skip
     const now = Date.now()
     await loaded()
     if (Date.now() - now < 20) {
-      this.oneShots[url].start()
+      this.oneShots[key].start()
     }
   }
 
+  async playGameOver() {
```

Comment:
this is nice. in #94 I was getting frustrated trying to make it look smooth at the end when a body turns into a star and it made me realize we should probably just have a dedicated game over sequence where we do all of it rather than try to integrate it smoothly into game play. This function could be where it's expanded for more happening than just music.